skimage/exposure/exposure.py

```diff
@@ -348,6 +348,8 @@ def rescale_intensity(image, in_range='image', out_range='dtype'):
 
     image = np.clip(image, imin, imax)
 
+    if imin == imax:
+        return image
     image = (image - imin) / float(imax - imin)
     return np.array(image * (omax - omin) + omin, dtype=dtype)
```

Comment:
can you please make this `asarray` instead of `np.array`, it may avoid a copy if the image is already a float.

Reply:
Sorry. I was referring to line 354, the final line.

Reply:
Okay, sounds good. Should I still return the original image if max == min? Or should I just skip the division on line 353 if max != min?

Reply:
Just skip the division part.

Reply:
I want to return omin if the image is a constant. Get it?

Reply:
Yep, got it